internal/config/flags.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/version"
+	"github.com/Unleash/unleash-client-go/v3"
+	"github.com/rs/zerolog"
+)
+
+func FeatureEnabled(name string) bool {
+	// When unleash is not configured, all features are enabled by default.
+	if !config.Unleash.Enabled {
+		return true
+	}
+
+	return unleash.IsEnabled(name)
```

Comment:
Also, we shall probably pass a context with some user info, so we can use the rollout strategies (like A-B testing).

https://github.com/Unleash/unleash-client-go#unleash-context

Reply:
Yeah I did not know what strategies are used on our instances, so I will simply use orgID for the "userId" in the context for the initial version.